mcp-server/README.md

```diff
@@ -0,0 +1,107 @@
+# Wren MCP Server
+
+The **Wren MCP Server** is a **Model Context Protocol (MCP) server** that provides tools for interacting with **Wren Engine** to facilitate AI agent integration.
+
+## Requirements
+
+Before setting up the Wren MCP Server, ensure you have the following dependency installed:
+
+- **[uv](https://docs.astral.sh/uv/getting-started/installation/#installing-uv)** - A fast and efficient Python package manager.
+
+## Environment Variables
+
+The server requires the following environment variables to be set:
+
+| Variable | Description |
+|----------|------------|
+| `WREN_URL` | The URL of the **Wren Ibis server**. |
+| `CONNECTION_INFO_FILE` | The path to the **required connection info file**. |
```

Comment:
We should specify that the connection information is stored in a JSON file and provide an example, as shown below

```json 
{
    "connectionInfo": {
        "host": "docker.for.mac.localhost",
        "port": "5432",
        "user": "test",
        "password": "test",
        "database": "test"
    }
}

```